hack/app_sre_pr_check.sh

```diff
@@ -5,22 +5,25 @@ set -exv
 trap "rm -f sorted-before*.yaml.tmpl sorted-after*.yaml.tmpl" EXIT
 
 PROMTOOL_IMAGE=quay.io/prometheus/prometheus
-CONTAINER_ENGINE=docker
+
+# Image that contains python, yaml & co
+PYTHON_IMAGE=quay.io/redhat-services-prod/openshift/boilerplate:latest
```

Comment:
Open for alternative suggestions for a smaller image. I think we're in a good place with boilerplate:latest though.